src/SDKs/HDInsight/DataPlane/Microsoft.Azure.HDInsight.Job/Microsoft.Azure.HDInsight.Job.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.HDInsight.Job</PackageId>
+		<Description>Azure HDInsight Job SDK Library</Description>
+		<AssemblyName>Microsoft.Azure.HDInsight.Job</AssemblyName>
+		<Version>3.0.0-preview</Version>
```

Comment:
We recommend starting `preview` versions from `0.9.0-preview`

Reply:
@dsgouda , please notice this is not a new package. We are trying to move from old Hyak-based jobs SDK to Swagger-based jobs SDK. You can find the old package [here](https://www.nuget.org/packages/Microsoft.Azure.Management.HDInsight.Job/), its version is `2.0.7`. That is the reason we want to make version of this package starts from `3.0.0-preview`. Does it make sense?

Reply:
Appreciate the info